Libs/DICOM/Widgets/ctkDICOMTableView.h

```diff
@@ -76,47 +76,53 @@ class CTK_DICOM_WIDGETS_EXPORT ctkDICOMTableView : public QWidget
    * @brief Setting the ctkDICOMDatabase which shall be queried
    * @param dicomDataBase the underlying database
    */
-  void setDicomDataBase(ctkDICOMDatabase* dicomDatabase);
+  Q_INVOKABLE void setDicomDataBase(ctkDICOMDatabase* dicomDatabase);
 
   /**
    * Setting the table name which shall be used for the database query
    * @param tableName the name of the database table
    */
-  void setQueryTableName(const QString &tableName);
+  Q_INVOKABLE void setQueryTableName(const QString &tableName);
```

Comment:
Why not property instead of invokable? (this question applies to all for all other methods that could be exposed to Python as a property instead)

Reply:
They could be, yes...